src/components/skeleton/Footer.vue

```diff
@@ -0,0 +1,54 @@
+<template>
+  <v-footer dark bottom padless>
+    <v-row >
+      <v-col class="pa-0" cols=12>
+        <v-card class="flex cyan darken-4 pa-5" flat tile>
+          <v-row align="center" class="ma-2">
+            <img
+              style="max-width: 160px; max-height: 50px"
+              class="ma-2"
+              :src="require('../../../public/images/couperin-text.png')"
+              alt="couperin-img"
+            />
+            <img
+              style="max-width: 150px; max-height: 50px"
+              class="ma-2"
+              :src="require('../../../public/images/counter-text.png')"
+              alt="counter-img"
+            />
+            <img
+              style="max-width: 70px; max-height: 70px"
+              class="ma-2"
+              :src="require('../../../public/images/cnrs-logo.svg')"
+              alt="cnrs-img"
+            />
+            <img
+              style="max-width: 170px; max-height: 170px"
+              class="ma-2"
+              :src="require('../../../public/images/ministry-text.png')"
+              alt="ministry-img"
+            />
```

Comment:
An array with `v-for` will be more readable (like in `ApplicationsMenu.vue` btw)